PR_Standard.md

```diff
@@ -0,0 +1,34 @@
+# Description
```

Comment:
Does this file need to be deleted? It looks like you have 2 copies of the template file.